build_tests_bazel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Where is the `build_tests_bazel.sh` used?

Reply:
i think we need to create a new GH-Action for it